File: app/src/main/java/com/dominikgaller/alpinebooking/booking/core/inport/result/RequestTourBookingResult.java
package com.dominikgaller.alpinebooking.booking.core.inport.result;

/**
 * Result returned after a successful RequestTourBooking use case execution.
 *
 * <p>Contains only standard-library types — no domain value objects.
 *
 * <p>SDD: See {@code documentation/ports/request-tour-booking.inport.spec.md}, section 2.2.
 */
public record RequestTourBookingResult(
        String bookingId,
        String status
) {}

File: src/main/kotlin/com/example/contractmanagement/booking/core/inport/result/RequestTourBookingResult.kt
package com.example.contractmanagement.booking.core.inport.result

/**
 * Output of the RequestTourBooking use case (UC01).
 *
 * SDD: see `documentation/ports/request-tour-booking.inport.spec.md` § 2.2.
 */
data class RequestTourBookingResult(
    val bookingId: String,
    val status: String,
)
